### PR TITLE
Add availability in numbers section

### PR DESCRIPTION
**Pull request by mebjas**
Points:
 - 9s of availability with example of 99.9 and 99.99 percentage
 - Difference in availability with components in sequence or parallel

### CODE REVIEW

**Comment by donnemartin** [comment]
Hi @mebjas this looks like a solid contribution, sorry I've been super busy and haven't had a chance to review it yet.

File: README.md [hunk 1]
```diff
@@ -527,6 +527,53 @@ This topic is further discussed in the [Database](#database) section:
...
+
+Given that Foo and Bar are independent, it's their probablity to be up together. 
+
+For two components with `99.9%` availability, total availability in sequence comes out to be `99.8%`. 
+
+#### Connected in Parallel
+The overall availability increases when two components with availability < 100% are connected in parallel. For two components `Foo` and `Bar` connected in parallel, the total availability comes out to be:
+
+```
+Availability (Total) = (1 - Availability (Foo)) * (1- Availability (Bar))
```
Comment (garukun) [inline README.md]: I think this math needs one more `1 - ` in the front, e.g. 

```
Availability (Total) = 1 - (1 - Availability (Foo)) * (1- Availability (Bar))
```

Comment (mebjas) [reply README.md]: oops, yes! Thanks @garukun

Comment (mebjas) [reply README.md]: Fixed

**Comment by mebjas** [comment]
> Hi @mebjas this looks like a solid contribution, sorry I've been super busy and haven't had a chance to review it yet.

Sure np

**Comment by mebjas** [comment]
@donnemartin - gentle reminder 2 :) Don't mind!

File: README.md [hunk 2]
```diff
@@ -527,6 +527,53 @@ This topic is further discussed in the [Database](#database) section:
 * [Master-slave replication](#master-slave-replication)
 * [Master-master replication](#master-master-replication)
 
+### Availability in numbers
+Availability is generally quanitifed with terms like uptime and downtime as percentage of time the service guarenteed was available or not respectively. This is popularly measured in number of 9s, for example as 99.9% or 99.99% availability is described as three 9s of availability and 4 9s respectively.
```
Comment (yamenfyi) [inline README.md]: quantified **

File: README.md [hunk 3]
```diff
@@ -527,6 +527,53 @@ This topic is further discussed in the [Database](#database) section:
...
+
+Here's what the numer transaltes to with respect to uptime:
```
Comment (yamenfyi) [inline README.md]: the numbers translate to **

**Comment by mebjas** [comment]
Thanks @yamenfyi - both comments resolved.

**Comment by donnemartin** [comment]
@mebjas @yamenfyi @garukun @IAmPramod thank you!

**Comment by mebjas** [comment]
Thanks

**Pull request merged**